File: Java/src/main/java/com/contidio/sdk/exceptions/InternalErrorBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class InternalErrorBackendException extends BackendException {
	public InternalErrorBackendException(String message) {
		super(BackendErrorCode.GENERIC_ERROR, message, "InternalErrorBackendException", HttpStatus.SC_INTERNAL_SERVER_ERROR);
	}	
}

File: .NET Core/Exceptions/InternalErrorBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class InternalErrorBackendException : BackendException
    {
        public InternalErrorBackendException(string message)
            : base(BackendErrorCode.GENERIC_ERROR, message, "InternalErrorBackendException", HttpStatusCode.InternalServerError)
        {
        }
    }
}
